src/libraries/System.Security.Cryptography.Cose/tests/CoseSign1MessageTests.Sign.cs

```diff
@@ -20,7 +22,7 @@ public void SignVerifyECDsa(ECDsaAlgorithm algorithm)
             ECDsa ecdsa = ECDsaKeys[algorithm];
             HashAlgorithmName hashAlgorithm = GetHashAlgorithmNameFromCoseAlgorithm((int)algorithm);
 
-            byte[] coseMessageBytes = CoseSign1Message.Sign(s_sampleContent, ecdsa, hashAlgorithm);
+            ReadOnlySpan<byte> coseMessageBytes = CoseSign1Message.Sign(s_sampleContent, ecdsa, hashAlgorithm);
```

Comment:
The tests that aren't using the abstract Sign method should either use it, or move to a different class.   (I assume this was just a missed case of changing over to the abstract method.  And the local variable can still be `byte[]`)